runtime/ejs-init.c

```diff
@@ -219,6 +222,11 @@ _ejs_init(int argc, char** argv)
     GLOBAL_METHOD(encodeURI);
     GLOBAL_METHOD(encodeURIComponent);
 
+    GLOBAL_METHOD(clearInterval);
```

Comment:
I think adding these to a method called `_ejs_timers_init` (inside `ejs-timers.c`) might not be a bad idea.  The other global methods could/should be moved as well (not in this PR :)